Sources/Danger/BitBucketServerDSL.swift

```diff
@@ -395,3 +395,7 @@ extension BitBucketServer {
         public let type: String?
     }
 }
+
+public var bitbucketServer: BitBucketServer! {
+    return DangerRunner.shared.dsl.bitbucketServer
+}
```

Comment:
<!--
  0 failure: 
  1 warning:  File should conta...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/BitBucketServerDSL.swift;
  Line: 401;
-->

- :warning: File should contain 400 lines or less: currently contains 401 (`file_length`)